helpers/collaborators.ts

```diff
@@ -0,0 +1,44 @@
+import * as Octokit from '@octokit/rest';
```

Comment:
I'm not 100% sure if this is the correct way of getting access to Octokit's types - but it seems to work? 🤷‍♂